modules/ROOT/pages/replay-transmissions.adoc

```diff
@@ -0,0 +1,69 @@
+= Replaying Transmissions
+
+Transmissions can fail due to data validation, mapping, or connectivity issues. If a transmission fails after an Anypoint Partner Manager endpoint secures it, you can replay the transmission.
+
+You can also replay transmissions that processed successfully. You might want to do this if:
+
+* The translated payload is missing a field. 
++
+In this scenario, you can upload a new translation map and replay the original transmission so that it goes through the updated map.
+* The target system or partner that received the original translated payloads lost data due to a system issue and requested that it be resent. 
+
+Replaying a transmission regenerates the transmission source and reprocesses the original transmission as a new transmission. You can replay both inbound and outbound transmissions.
+
+The following rules apply to replaying transmissions:
+
+* You can replay any original transmission that has a status of `Failed` or `Delivered`, except for inbound transmissions that failed AS2 security validation.
+* The maximum number of transmissions that you can replay in a single request is 100. 
+* Partner Manager links replayed transmission instances to the original transmission record.
++
+* When a transmission is replayed. Partner Manager sets its transmission status to `Replaying`. Once the replay action completes, Partner Manager shows the date of the last replay.
++
+To check the status of a replayed transmission, refresh the *Activity* screen and locate the transmission.
++
+* Partner Manager processes the transmission payload through the active version of the message flow. Therefore, the new transmission uses the current map and endpoint configurations for resolution and routing.
++
+If Partner Manager doesn't find an active message flow for the transmission or message, it returns an error. 
++
+* Partner Manager doesn't check for duplicate control numbers when it replays inbound transmissions.
+* Partner Manager sends acknowledgments for replayed inbound transmissions only if the acknowledgement rules changed in the message flow definition from the previous transmission or if the acknowledgment outcome is different from the previous attempt.
```

Comment:
Please change to: 
Partner Manager sends acknowledgments for replayed inbound transmissions based on the partner or message flow configurations at the time of replaying.